src/components/PrivateRoute/PrivateRoute.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import { Route } from 'react-router-dom'
+import { Redirect } from 'react-router'
+import { useCustomHook } from '../../context/UserContext'
+
+export const PrivateRoute = ({ children, ...rest }) => {
+    const { user } = useCustomHook();
+    return (
+     
+    )
```

Comment:
Hey @Dylan-Greathouse! Looks like you forgot to add in the private route return logic here and it's causing your Netlify build to fail so I'm not able to view any of the app. Let me know when this is sorted and I'll take another look!